security/pass-otp/Portfile

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8::et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        tadfisher pass-otp 1.1.1 v
+maintainers         {loicp.eu:loic-github @lpefferkorn}
```

Comment:
Can openmaintainer be added?

Reply:
We indeed generally prefer that ports be `openmaintainer` unless there's good reason not to do that.

Reply:
Will add!